test/bootstrap_form_with_radio_button_test.rb

```diff
@@ -0,0 +1,138 @@
+    # end
+    #
+    # test 'collection_radio_buttons renders label defined by lambda correctly' do
+    #   collection = [Address.new(id: 1, street: 'Foobar')]
+    #   expected = %{<div class="form-group"><label class="form-control-label" for="user_misc">This is a radio button collection</label><div class="radio"><label for="user_misc_1"><input id="user_misc_1" name="user[misc]" type="radio" value="1" /> rabooF</label></div><span class="form-text text-muted">With a help!</span></div>}
+    #
+    #   assert_equivalent_xml expected, @form_with_builder.collection_radio_buttons(:misc, collection, :id, lambda { |a| a.street.reverse }, label: 'This is a radio button collection', help: 'With a help!')
+    # end
+    #
+    # test 'collection_radio_buttons renders value defined by lambda correctly' do
+    #   collection = [Address.new(id: 1, street: 'Foobar')]
+    #   expected = %{<div class="form-group"><label class="form-control-label" for="user_misc">This is a radio button collection</label><div class="radio"><label for="user_misc_address_1"><input id="user_misc_address_1" name="user[misc]" type="radio" value="address_1" /> Foobar</label></div><span class="form-text text-muted">With a help!</span></div>}
+    #
+    #   assert_equivalent_xml expected, @form_with_builder.collection_radio_buttons(:misc, collection, lambda { |a| "address_#{a.id}" }, :street, label: 'This is a radio button collection', help: 'With a help!')
+    # end
+    #
+    # test 'collection_radio_buttons renders multiple radios with label defined by lambda correctly' do
+    #   collection = [Address.new(id: 1, street: 'Foo'), Address.new(id: 2, street: 'Bar')]
+    #   expected = %{<div class="form-group"><label class="form-control-label" for="user_misc">Misc</label><div class="radio"><label for="user_misc_1"><input id="user_misc_1" name="user[misc]" type="radio" value="1" /> ooF</label></div><div class="radio"><label for="user_misc_2"><input id="user_misc_2" name="user[misc]" type="radio" value="2" /> raB</label></div></div>}
+    #
+    #   assert_equivalent_xml expected, @form_with_builder.collection_radio_buttons(:misc, collection, :id, lambda { |a| a.street.reverse })
+    # end
+    #
+    # test 'collection_radio_buttons renders multiple radios with value defined by lambda correctly' do
+    #   collection = [Address.new(id: 1, street: 'Foo'), Address.new(id: 2, street: 'Bar')]
+    #   expected = %{<div class="form-group"><label class="form-control-label" for="user_misc">Misc</label><div class="radio"><label for="user_misc_address_1"><input id="user_misc_address_1" name="user[misc]" type="radio" value="address_1" /> Foo</label></div><div class="radio"><label for="user_misc_address_2"><input id="user_misc_address_2" name="user[misc]" type="radio" value="address_2" /> Bar</label></div></div>}
+    #
+    #   assert_equivalent_xml expected, @form_with_builder.collection_radio_buttons(:misc, collection, lambda { |a| "address_#{a.id}" }, :street)
+    # end
```

Comment:
commented out tests?

Reply:
Thanks for catching this. I will uncomment, but I will also have to figure out what to do about the DOM IDs mentioned above.